Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -185,7 +185,7 @@ public function build(array $buildSubject)
         }
 
         $requestBody['additionalData']['allow3DS2'] = true;
-
+        //$requestBody['additionalData']['RequestedTestAcquirerResponseCode'] = 2;
```

Comment:
```suggestion
```

Reply:
oh no ! that was from testing amazonpay. Thanks @acampos1916